wagtail/core/views.py

```diff
@@ -14,7 +14,7 @@ def serve(request, path):
         raise Http404
 
     path_components = [component for component in path.split('/') if component]
-    page, args, kwargs = site.root_page.specific.route(request, path_components)
+    page, args, kwargs = site.root_page.localized.specific.route(request, path_components)
```

Comment:
Very nice! Was expecting to find all kinds of fancy new routing logic here, so it's really good to see this condensed into something so minimal and self-explanatory :-)